### DISCUSSION TITLE
Mastodon, Fediverse or other social media links in profile

### DISCUSSION THREAD

**Discussion by pragdevops** [community/community#43466] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be great to be able to add more (optional) links to external profiles to the GitHub profile, like links to LinkedIn, Xing, Mastodon etc.

**Comment by radar-blip** [comment]
This should be in your settings now:

![socialaccts](https://user-images.githubusercontent.com/39196767/216243810-142cfcc1-4b52-4542-9435-8b3938bebd24.png)
